go.mod

```diff
@@ -38,12 +38,12 @@ require (
 	github.com/grafana-tools/sdk v0.0.0-20220919052116-6562121319fc
 	github.com/jarcoal/httpmock v1.3.0
 	github.com/jinzhu/copier v0.3.5
-	github.com/layer5io/gowrk2 v0.0.0-20191111234958-a4c9071c0f87
+	github.com/layer5io/gowrk2 v0.6.1
```

Comment:
@abdullah1308, will you confirm if we're going to have an issue with this newer version of gowrk2 and fortio?

Reply:
Ok, I see that this is @abdullah1308's commit. 👍